tasks/main.yml

```diff
@@ -143,7 +143,7 @@
     - vault_gkms_credentials_src_file | length > 0
 
 - name: "Copy over GCP Credentials for Auto Unseal"
-  copy:
+  template:
```

Comment:
Please revert this because it breaks task.

There is no templating, so if necessary add lint exclude.

Reply:
Hmm, how does it break? Are you sure we can't fix it to function as a template? It's better if we can fix it rather than adding a lint exclude.

Reply:
Ok, I'll try to explain in detail.

At beginning the role was just copying 'vault_gkms_credentials_src_file' from control machine to managed host(s).

Later I implemented option to load this from variable in non breaking way. At first previous task check if there is file, if yes, then it will slurp its content into variable. 

So this variable can be set by inventory or slurped from ansible control machine.

In copy task we just take this content and copy it to managed host(s).

Ansible-lint is mistaken in detecting this as tempalte action. Moreover there is no elegant option to use variable in template module. Only option would by introduce template with just one variable inside it, what I think is serving to linter in way that is making code more  spaghetti then it should be.

Reply:
I forgot to mention that template module doesn't have parameter content.

Reply:
Ok got it, it's a bit ironic that the linter recommends using the template module and then doesn't complain that the content parameter is invalid for the template module.

As the template module always requires a file as a input I wonder if it would be good to use the lineinfile module to write the variable to a file? Since it can write the content of a variable to a file and it can create files if don't exist. We could then just use the regex in the lineinfile so that it will overwrite the whole file.

Reply:
Using lineinfile could be the solution.

Reply:
So I tried a few approaches, with using `lineinfile` `blockinfile` `replace` but was unable to get them to work correctly for this use case.
Only option left would be to create a template that just contains that single variable, but that's a silly solution imho.